src/variant_matrix/VariantMatrixViews.cc

```diff
@@ -11,8 +11,11 @@ namespace
             {
                 throw std::out_of_range("row index out of range");
             }
```

Comment:
These functions have too much repetition and are only difference in their const-ness.  Reduce/DRY.

Reply:
I'm going to let this slide for now.